Support/Scripts/run-lldb-tests.sh

```diff
@@ -65,17 +71,19 @@ while test $# -gt 0; do
     --strace) opt_strace=true;;
     --use-lldb-server) opt_use_lldb_server=true;;
     --lldb-tests) shift
-	    LLDB_TESTS="$1";;
+      LLDB_TESTS="$1";;
     --target) shift
-	    TARGET="$1";;
-    --platform) shift
-	    PLATFORM="$1";;
+      TARGET="$1";;
+    --platform) PLATFORM=1;;
+    --pudb) PUDB="-m pudb";;
+    --dostep-extra-args) shift
+      DOSTEP_EXTRA_ARGS="$1";;
     *) die "Unknown option \`$1'.";;
   esac
   shift
 done
 
-TARGET="${TARGET-${CIRCLE_JOB}}"
+TARGET="${TARGET-${CIRCLE_JOB-}}"
```

Comment:
Don’t we need to have at least one of these two set?

Reply:
No, dostep.py defaults to the value derived from the python `sys.platform` or similar for the host given an unset `TARGET`.

Reply:
Nice. If that works I'm ok with this then.